recipe/meta.yaml

```
@@ -59,3 +55,4 @@ extra:
     - marcelotrevisani
     - xhochy
     - mbargull
+    - beckermr
```

@kkraus14 is it ok if we add you to the maintainers here?

```suggestion
    - beckermr
    - kkraus14
```